api-design-guidelines/index.md

```diff
@@ -93,7 +93,9 @@ is printed.
       great summary.
 
     * **Use a single sentence fragment** if possible, ending with a
-      period.  Do not use a complete sentence.
+      period.  Do not use a complete sentence.  A summary is like 
+      a dictionary definition; it is shown in the context of the 
+      API it describes.
```

Comment:
I have concerns about talking about where the summary “is shown.”  This will only (necessarily) apply to Xcode users.  I wonder if this would handle the confusion:

> **Make the summary read like a dictionary definition**, using a single sentence fragment and ending with a period. Do not use a complete sentence.